chart/epinio/values.yaml

```diff
@@ -169,7 +169,6 @@ s3gw:
     enabled: false
   serviceName: s3gw
   useExistingSecret: true
-  defaultUserCredentialsSecret: s3gw-creds
```

Comment:
I would leave this, no need to remove, when we will retry a bump we will already have this.